distribution/scripts/jmeter/run-backend-tests.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
+# Copyright (c) 2019, WSO2 Inc. (http://wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+# Run Backend Performance Tests
+# ----------------------------------------------------------------------------
+
+script_dir=$(dirname "$0")
+# Execute common script
+. $script_dir/perf-test-common.sh
+
+function initialize() {
+    export backend_host=$(get_ssh_hostname $backend_ssh_host)
+}
+export -f initialize
+
+declare -A test_scenario0=(
+    [name]="netty-backend-http"
+    [display_name]="Netty-Backend-HTTP"
+    [description]="This test directly invokes the Netty Backend Service through HTTP protocol"
+    [jmx]="backend-test.jmx"
+    [protocol]="http"
+    [path]="/echo/1.0.0"
+    [use_backend]=false
+    [skip]=false
+)
+
+declare -A test_scenario1=(
+    [name]="netty-backend-https"
+    [display_name]="Netty-Backend-HTTPS"
+    [description]="This test directly invokes the Netty Backend Service through HTTPS protocol"
+    [jmx]="backend-test.jmx"
+    [protocol]="https"
+    [path]="/echo/1.0.0"
+    [use_backend]=false
+    [skip]=false
+)
+
+function before_execute_test_scenario() {
+    local service_path=${scenario[path]}
+    local protocol=${scenario[protocol]}
+    jmeter_params+=("host=$backend_host" "port=8688" "path=$service_path")
+    jmeter_params+=("payload=$HOME/${msize}B.json" "response_size=${msize}B" "protocol=$protocol")
+}
+
+function after_execute_test_scenario() {
```

Comment:
I think that this function is not required.

Reply:
This function is being referred from perf-test-common script. Cannot remove it. Hence added this method as an empty method returning true.

Reply:
That's okay. I think you can just add `return 0`, which is more clear than using `:`